Calculator.py

```diff
@@ -17,6 +17,9 @@ def run_operation(number_1,  number_2, operation):
     elif operation == '/':
         print('{} / {} = '.format(number_1, number_2), end = '')
         return number_1 / number_2
+    elif operation == '%':
+        print('{} % {} = '.format(number_1, number_2), end = '')
+        return number_1 % number_2
     else:
         raise Exception('This operator is invalid. Try running the program again.')
```

Comment:
good error checking perhaps add this exception before a user enters numbers.

Reply:
Agreed. Adding internal error handling in the event the user passes non-int values, allowing the program to continue running by prompting the user to re-enter with a valid value, would be ideal.